microservices/blog/migrations/1693301991584-init.ts

```diff
@@ -8,7 +8,7 @@ export default class init1693301991584 implements MigrationInterface {
      * Tables
      */
     await queryRunner.query(
-      `CREATE TABLE "article" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "userId" character varying(36), "title" character varying(255) NOT NULL, "alias" character varying(255) NOT NULL, "description" character varying(255) NOT NULL, "content" text NOT NULL DEFAULT '', "publishDate" TIMESTAMP, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "article(uq):alias" UNIQUE ("alias"), CONSTRAINT "article(pk):id" PRIMARY KEY ("id"))`,
+      `CREATE TABLE "article" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "userId" character varying(36), "title" character varying(255) NOT NULL, "alias" character varying(255) NOT NULL, "description" character varying(255) NOT NULL, "content" text NOT NULL DEFAULT '', "publishDate" TIMESTAMP,  "extra" json NOT NULL DEFAULT '{}', "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "article(uq):alias" UNIQUE ("alias"), CONSTRAINT "article(pk):id" PRIMARY KEY ("id"))`,
```

Comment:
Should be params instead of extra

Reply:
it's not params field, that field uses to store some extra data